apps/desktop-wallet/src/features/walletConnect/walletConnectContext.tsx

```diff
@@ -410,7 +410,11 @@ export const WalletConnectContextProvider: FC = ({ children }) => {
             const p = request.params as ApiRequestArguments
 
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            const call = (client.explorer as any)[`${p.path}`][`${p.method}`] as (...arg0: any[]) => Promise<any>
+            const call = (throttledClient.explorer as any)[`${p.path}`][`${p.method}`] as (
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
+              ...arg0: any[]
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            ) => Promise<any>
```

Comment:
We should look into whether this can eventually be simplified, i rly don't like it, but I also don't wanna touch it.